### PR TITLE
11.0 Small docstring improvements

### PR DESCRIPTION
**Pull request by guewen**

### CODE REVIEW

File: queue_job/models/base.py [hunk 1]
```diff
@@ -8,7 +8,11 @@
 
 
 class Base(models.AbstractModel):
-    """ The base model, which is implicitly inherited by all models. """
+    """ The base model, which is implicitly inherited by all models.
```
Comment (pedrobaeza) [inline queue_job/models/base.py]: Shouldn't you remove the initial space the same as you have made with the other docstrings?

File: queue_job/models/queue_job.py [hunk 2]
```diff
@@ -22,7 +22,7 @@ def channel_func_name(model, method):
 
 
 class QueueJob(models.Model):
-    """ Job status and result """
+    """ Model storing the jobs to be executed. """
```
Comment (pedrobaeza) [inline queue_job/models/queue_job.py]: The same here

File: queue_job/models/base.py (hunk 1 above)
Comment (guewen) [reply queue_job/models/base.py]: sure, thanks for noticing

File: queue_job/controllers/main.py [hunk 3]
```diff
@@ -27,7 +27,7 @@
 class RunJobController(http.Controller):
 
     def _load_job(self, env, job_uuid):
-        """ Reload a job from the backend """
+        """Reload a job from the backend"""
```
Comment (simahawk) [inline queue_job/controllers/main.py:30]: should end w/ dot :stuck_out_tongue_winking_eye:

**Review by simahawk — APPROVED** [review]

**Comment by pedrobaeza** [comment]
There's seems to be a warning on runbot:
```
2018-07-30 07:54:23,944 165 WARNING openerp_test odoo.addons.base.module.module: docutils' system message present: <system_message backrefs="id10" level="1" line="3" source="<string>" type="INFO"><paragraph>Duplicate explicit target name: "#69".</paragraph></system_message
```

**Comment by guewen** [comment]
> There's seems to be a warning on runbot:

not anymore :)

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**